inginious/frontend/submission_manager.py

```diff
@@ -284,6 +284,16 @@ def add_job(self, task, inputdata, debug=False):
         inputdata["@random"] = states["random"] if "random" in states else []
         inputdata["@state"] = states["state"] if "state" in states else ""
 
+        # Send LTI information to the client except "consumer_key"
+        lti_info = self._user_manager.session_lti_info()
```

Comment:
This may return ``None`` if the session is not identified as an LTI one. Putting ``or {}`` at the end of the line should fix this issue I guess.

Reply:
Add "if clause" to check if lti_info is available
```suggestion
        lti_info = self._user_manager.session_lti_info()
        if lti_info:
```